### PR TITLE
1384999, update real-time chat with SignalR 2.

### PR DESCRIPTION
**Pull request by john-par**
Fixes #1384999 

Update for task to use .NET tutorial template.

### CODE REVIEW

**Comment by john-par** [comment]
@tdykstra , this PR is ready for your review. Please let me know what you'd like changed.

Also, please note that I did not include a [**Get the code**](https://review.docs.microsoft.com/en-us/help/contribute/templates/dotnet/tutorial-template-dotnet?branch=master) section because I could find no samples on GitHub that match this article. These are the repos I looked in:

*  [SignalR/SignalR](https://github.com/SignalR/SignalR/tree/master/samples/Microsoft.AspNet.SignalR.Samples)
* [aspnet/SignalR-samples](https://github.com/aspnet/SignalR-samples)

I am assuming that matching samples don't exist because of the age of the article, but if you know of samples you'd like me to link to, let me know.

File: aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr.md [hunk 1]
```diff
@@ -1,209 +1,182 @@
 ---
 uid: signalr/overview/getting-started/tutorial-getting-started-with-signalr
-title: "Tutorial: Getting Started with SignalR 2 | Microsoft Docs"
+title: "Real-time chat with SignalR 2 | Microsoft Docs"
 author: pfletcher
-description: "This tutorial shows how to use SignalR to create a real-time chat application. You will add SignalR to an empty ASP.NET web application and create an HTML pa..."
+description: "This tutorial shows how to use SignalR to create a real-time chat application. You will add SignalR to an empty ASP.NET web application and..."
```
Comment (tdykstra) [inline aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr.md]: Something's missing here, I would just drop the and + ellipsis, and terminate the sentence with a period.

**Comment by john-par** [comment]
@tdykstra  all of your feedback has been incorporated.

**Comment by tdykstra** [comment]
@john-par Are your PRs in which you added an "all feedback addressed" comment ready to merge? (I assume so, just checking before I merge them.)

**Pull request merged**